js/search.js

```diff
@@ -1307,14 +1309,12 @@
 		// number
 		// buf += '<span class="col numcol">' + (pokemon.num >= 0 ? pokemon.num : 'CAP') + '</span> ';
 		var tier;
-		if (this.gen < 7) {
-			tier = pokemon.num;
-		} else if (pokemon.tier) {
-			tier = pokemon.tier;
+		if (pokemon.tier) {
+			tier = Dex.getTier(pokemon, this.gen, this.isDoubles);
 		} else if (pokemon.forme && pokemon.forme.endsWith('Totem')) {
-			tier = Dex.getTemplate(pokemon.species.slice(0, (pokemon.forme.startsWith('Alola') ? -6 : pokemon.baseSpecies.length + 1))).tier;
+			tier = Dex.getTier(BattlePokedex[toId(pokemon.species.slice(0, (pokemon.forme.startsWith('Alola') ? -6 : pokemon.baseSpecies.length + 1)))], this.gen, this.isDoubles);
 		} else {
-			tier = Dex.getTemplate(pokemon.baseSpecies).tier;
+			tier = Dex.getTier(BattlePokedex[toId(pokemon.baseSpecies)], this.gen, this.isDoubles);
 		}
```

Comment:
I seem to have missed this, but the point of `getTier` was to be able to replace these 7 lines with a single call to `getTier`, which seems to not be what's going on here.